.ci/pipelines/release_zeebe_java8.groovy

```diff
@@ -122,6 +122,10 @@ spec:
                 container('maven') {
                     sh '.ci/scripts/release/github-release.sh'
                 }
+
+                container('golang') {
+                    sh '.ci/scripts/release/post-release-go.sh'
+                }
```

Comment:
I take it there was no `post-github` step before, since this a new addition?

Reply:
Yes. To be honest, it doesn't matter as we deleted the java 8 job, it's just an old remnant in the 1.2 branch.